bindings/rust/bench/use-awslc-rustls.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# sets bench crate to use aws-lc-rs instead of ring for rustls
+
+set -e
+
+# go to bench directory
+pushd "$(dirname "$0")" > /dev/null
+bench_dir="$(pwd)"
+
+# clone rustls to bench/target/rustls and checkout compatible version
+rm -rf target/rustls
+git clone https://github.com/rustls/rustls target/rustls
+cd target/rustls
+git checkout 'v/0.21.5'
+
+# go to dir with rustls crate
+cd rustls
+rustls_dir="$(pwd)"
+
+# change rustls to use aws-lc-rs
+sed -i 's|ring = .*|ring = { package = "aws-lc-rs" }|' Cargo.toml
```

Comment:
modifying source is pretty hacky. We should not be doing this

Reply:
There's no better way to test aws-lc-rs with rustls afaik. I tried for looking ways to patch sub-dependencies (like ring is for rustls in the case of the bench crate) with a package substitution, but I couldn't find any, especially since it is a pretty niche use case. aws-lc-rs itself uses the modifying source too: https://github.com/aws/aws-lc-rs/blob/1036316d5557b72e064948514d941dfc7afa78d7/aws-lc-rs/scripts/run-rustls-integration.sh